woocommerce/woocommerce-ios
# [Woo POS] Use WooFoundation.CurrencyFormatter to format POS product prices

#12705


## Description


Basic usage of `WooFoundation.CurrencyFormatter` to format POS product prices

## Testing instructions


- Enter POS experience
- Check that the product prices are showing the correct amount and currency symbol

## Screenshots


![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-23 at 11 51 02](https://github.com/woocommerce/woocommerce-ios/assets/6242034/7086c67c-c3c9-4e9a-9edb-91ebe4dee0e5)

---
